indexers/taurus/consensus/src/mappings/mappingHandlers.ts

```diff
@@ -1,3 +1,6 @@
+global.TextEncoder = require("util").TextEncoder;
+global.TextDecoder = require("util").TextDecoder;
```

Comment:
Why this?

Reply:
The environment SubQuery handler is running in, does not have TextEncoder&TextDecoder available, they needed to be imported.